src/wp-admin/includes/widgets.php

```diff
@@ -262,7 +284,7 @@ function wp_widget_control( $sidebar_args ) {
 				<button type="button" class="button-link widget-control-close"><?php _e( 'Done' ); ?></button>
 			</span>
 		</div>
-		<div class="alignright<?php if ( 'noform' === $has_form ) echo ' widget-control-noform'; ?>">
+		<div class="alignright<?php echo $noform_class; ?>">
```

Comment:
Why is the variable set way up on line 265? 
It's much less legible.

Reply:
Does match upstream, I can only think it was to assemble the PHP code together in one block.

Reply:
Discussed at PR review